### DISCUSSION TITLE
Scaling engines doesn't add enought weight.

### DISCUSSION THREAD

**Discussion by jeq1** [TweakScale/TweakScale#315] [Support]
I expect, just like scaling with fuel tanks, going from 1,25 to 2,5 weight should go up 8times, with engines weight gain is lot smaller, so scaling engines up feels cheating now.

**Comment by Lisias** [comment]
Nope.

Empirical research suggested that Engines's mass scales at at 2.5 exponent, not 3. Engines are not solid objects, they are pretty hollow inside (as the combustion chamber).
